pkg/kudo/operator.go

```diff
@@ -205,15 +222,74 @@ func (operator Operator) Uninstall() error {
 		Delete(operator.Operator.Name, &options)
 	if err != nil {
 		return fmt.Errorf(
-			"failed to delete Operator %s in namespace %s: %w",
+			"failed to delete Operator %s in namespace %s: %v",
 			operator.Operator.Name,
 			operator.Operator.Namespace,
 			err)
 	}
 
+	if timeout != 0 {
+		kudoV1beta1 := operator.client.Kudo.KudoV1beta1()
+		i := operator.Instance
+		ov := operator.OperatorVersion
+		o := operator.Operator
+		timeoutSeconds := timeout.Round(time.Second).Milliseconds() / 1000
+
+		err := waitForDeletion(kudoV1beta1.Instances(i.Namespace), i.ObjectMeta, timeoutSeconds)
+		if err != nil {
+			return err
+		}
+
+		err = waitForDeletion(kudoV1beta1.OperatorVersions(ov.Namespace), ov.ObjectMeta, 10)
```

Comment:
How did you determine 10 seconds here? Would that be a safe value on a heavily loaded cluster?
Let's make this a constant.

Reply:
This was just a guess, suggestions for a better value most welcome.
Changed to a const and added a comment.